cmd/app/link.go

```diff
@@ -173,14 +173,15 @@ func LinkExistingApp(ctx context.Context, clients *shared.ClientFactory, app *ty
 			Emoji: "warning",
 			Text:  "Warning",
 			Secondary: []string{
-				"Existing apps have manifests configured by app settings",
-				"Linking existing apps requires the manifest source to be " + config.ManifestSourceRemote.String(),
-				fmt.Sprintf(`Manifest source can be "%s" or "%s"`, config.ManifestSourceLocal.String(), config.ManifestSourceRemote.String()),
+				"Linking an existing app requires the app manifest source to be",
+				fmt.Sprintf("managed by %s.", config.ManifestSourceRemote.Human()),
 				" ",
-				fmt.Sprintf(style.Highlight(`Your manifest source is "%s"`), manifestSource.String()),
+				fmt.Sprintf(`App manifest source can be "%s" or "%s":`, config.ManifestSourceLocal.Human(), config.ManifestSourceRemote.Human()),
 				fmt.Sprintf("- %s: uses manifest from your project's source code for all apps", config.ManifestSourceLocal.String()),
 				fmt.Sprintf("- %s: uses manifest from app settings for each app", config.ManifestSourceRemote.String()),
 				" ",
+				fmt.Sprintf(style.Highlight(`Your manifest source is "%s"`), manifestSource.Human()),
+				" ",
```

Comment:
note: I tweaked the wording now that we've had some time to reflect on this. Once we default Bolt projects to remote manifest, this warning will no longer be shown, so it's not a long-lived change.

Reply:
Good question! I think the warning becomes less common after `bolt-install` and we could consider changing it to a message instead of a prompt.

The warning should be displayed when a project manages the app manifest (`manifest.source: "local"`) and an existing app is linked (`slack app link`). 

Right now, the warning prompts you to switch to a `remote` manifest source. I think we have an opportunity to make this more flexible. There are some cases where you may want to link an app while keeping your `local` manifest source.

Long story short, we'll want to think more about this as we make decisions around `bolt-install` 😆